src/ResourceManager/Compute/Commands.Compute/Commands.Compute.csproj

```diff
@@ -48,6 +48,9 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.Web.Administration, Version=7.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35">
```

Comment:
Microsoft.Web.Administration, Version=7.9.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=**MSIL**.

Doesn't look platform-specific to me... 
This change is harmless - the only thing it'll do is prevent the assembly from being copied. If the assembly cannot be found, it will have no effect.

Also note the corrected reference to SMA in src/ServiceManagement/ExpressRoute/Commands.ExpressRoute/Commands.ExpressRoute.csproj.